stores/member.ts

```
@@ -0,0 +1,78 @@
+      return arr.sort((a, b) => a.sort - b.sort)
+    },
+  },
+
+  actions: {
+    // 会员列表
+    async getMemberList(params: MemberReq) {
+      try {
+        const { data } = await https.post<MemberRes, MemberReq>('/member/list', params)
+        if (data.value.code === HttpCode.SUCCESS) {
+          this.memberListTotal = data.value.data.total
+          params.page === 1 ? this.memberList = data.value.data.list : this.memberList = this.memberList.concat(data.value.data.list)
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)